pkg/security/tests/testdata/exec-01_input_event.json

```diff
@@ -0,0 +1,142 @@
+{
+  "usr": {},
+  "evt": {
+    "name": "exec",
+    "category": "Process Activity",
+    "outcome": "Success"
+  },
+  "date": "0001-01-01T00:00:00Z",
+  "file": {
+    "path": "/bin/ls",
+    "name": "ls",
+    "inode": 2,
+    "uid": 0,
+    "gid": 0,
+    "modification_time": "1970-01-01T01:00:00+01:00",
+    "change_time": "1970-01-01T01:00:00+01:00",
+    "hash_state": "NoHash"
+  },
+  "process": {
+    "parent": {
+      "pid": 11,
+      "uid": 0,
+      "gid": 0,
+      "comm": "bash",
+      "credentials": {
+        "uid": 0,
```

Comment:
I know that from the serializer but maybe we could get rid of the empty values ?

Reply:
The JSON files have to be valid event ones for the backend CI job to be able to intake and process them. So, as it have to be compliant with the schema with all the required fields, IMHO we should stick to the default serializer behavior